src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlCheck.java

```diff
@@ -134,7 +134,7 @@
  * </p>
  * <ul>
  * <li>
- * {@code import.control.disallowed}
+ * {@code import.control.forbidden}
```

Comment:
this is breaking compatibility unfortunately. 
Lets keep it as it was before, it is very cryptic name that means not much, but users might use it.